active-directory-b2c-wpf/App.xaml.cs

```diff
@@ -13,16 +14,45 @@ namespace active_directory_b2c_wpf
     /// </summary>
     public partial class App : Application
     {
-        private static readonly string Tenant = "fabrikamb2c.onmicrosoft.com";
-        private static readonly string AzureAdB2CHostname = "fabrikamb2c.b2clogin.com";
+        /// <summary>
+        /// B2C tenant name
+        /// </summary>
+        private static readonly string TenantName = "fabrikamb2c";
+        private static readonly string Tenant = $"{TenantName}.onmicrosoft.com";
+        private static readonly string AzureAdB2CHostname = $"{TenantName}.b2clogin.com";
+
+        /// <summary>
+        /// ClientId for the application which initiates the login functionality (this app)  
+        /// </summary>
         private static readonly string ClientId = "841e1190-d73a-450c-9d68-f5cf16b78e81";
+
+        /// <summary>
+        /// Should be one of the choices on the Azure AD B2c / [This App] / Authentication blade
+        /// </summary>
         private static readonly string RedirectUri = "https://fabrikamb2c.b2clogin.com/oauth2/nativeclient";
```

Comment:
you could use AzureAdB2CHostname constant here

Reply:
This one turned out to be more complicated.  When figuring out this patch, I was actually using:

"com.MyCompany.MyApp://oauth/redirect";

But I didn't want to subject folks on this demo to that world quite yet, so I left it as is. Ultimately, I ended up not using this value myself (with 3.1) since it has to use the system browser now..   So WithDefaultRedirectUri() was where I ended up instead.